examples/binaries/constructor/src/arg.rs

```diff
@@ -0,0 +1,102 @@
+use alloc::{
+    string::{String, ToString},
+    vec::Vec,
+};
+use parity_scale_codec::{Codec, Decode, Encode};
+
+#[derive(Clone, Debug, Decode, Encode)]
+pub enum Arg<T: 'static + Clone + Codec> {
```

Comment:
I guess some doc-comments would be appreciated.

Reply:
Added minimalistic ones